### ISSUE TITLE
ktest uncaught exception

### ISSUE THREAD

**Issue by grosu** [kframework/k-legacy#519]
Arguments need -- instead of -, but better error message needed:

```
fslwork$ ktest tests/config.xml -programs=. -results=.
org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -programs=.
    at org.apache.commons.cli.Parser.processOption(Parser.java:363)
    at org.apache.commons.cli.Parser.parse(Parser.java:199)
    at org.apache.commons.cli.Parser.parse(Parser.java:85)
    at org.kframework.ktest.CmdArgs.CmdArgParser.<init>(CmdArgParser.java:81)
    at org.kframework.ktest.KTest.<init>(KTest.java:36)
    at org.kframework.ktest.KTest.main(KTest.java:107)
    at org.kframework.main.Main.main(Main.java:58)
Exception in thread "main" java.lang.NullPointerException
    at org.kframework.ktest.KTest.main(KTest.java:112)
    at org.kframework.main.Main.main(Main.java:58)
```

**Comment by traiansf** [comment]
What I did for RVPredict was to catch the exception and simply print the
message, which would be Unrecognized option: -programs=.
then to exit.

2014-05-30 5:25 GMT+03:00 Grigore Rosu notifications@github.com:

**Comment by daejunpark** [comment]
It seems to be same with the issue #485, and Omer gave a pull request #522.
Will it be also fixed with the pull request?

**Comment by osa1** [comment]
Yes.

**Comment by osa1** [comment]
Fixed with #522.
